CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Include stripped (non flatten) data into event (#377)
```

Comment:
Maybe documentation should be modified? Maybe the following sections are impacted by the new functionality:

https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#metadata-and-object-values
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#location-fields
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#time-fields
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#json-and-array-fields

(Not sure if I'm right. Should be checked)

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/393/commits/74cd861da01df386697e5fade1e3a14670a59677